code/src/components/AlbumCard.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Icons } from './Icons';
```

Comment:
Out of curiosity, why do you add { } here? Is it bc you import many icons?

Reply:
No, the number of icons does not affect how the import should be done. 
The reason why there are curly brackets there is because the component is being imported as a named component, and for named components you need to add `{}` around the name. It should actually be a default import/export though since there is only one thing to export in Icons.js, so will fix that. Good spot, thank you!